CPAC/resources/configs/pipeline_config_blank.yml

```diff
@@ -724,6 +724,9 @@ registration_workflows:
           #Input the index of which volume from the functional 4D timeseries input file you wish to use as the input for functional-to-anatomical registration.
           func_reg_input_volume: 0
 
+        # Independent of the above `reg_with_skull` option
```

Comment:
Just change this comment to "Mask the sbref that is created by the coregistration input prep nodeblocks above." or something similar.